diff --git a/node_modules/react-native-google-mobile-ads/ios/RNGoogleMobileAds/RNGoogleMobileAdsNativeModule.mm b/node_modules/react-native-google-mobile-ads/ios/RNGoogleMobileAds/RNGoogleMobileAdsNativeModule.mm
index 68668a6..25c64fb 100644
--- a/node_modules/react-native-google-mobile-ads/ios/RNGoogleMobileAds/RNGoogleMobileAdsNativeModule.mm
+++ b/node_modules/react-native-google-mobile-ads/ios/RNGoogleMobileAds/RNGoogleMobileAdsNativeModule.mm
@@ -103,6 +103,7 @@ - (instancetype)init {
           @"icon" : (nativeAd.icon && nativeAd.icon.imageURL != nil)
               ? @{@"scale" : @(nativeAd.icon.scale), @"url" : nativeAd.icon.imageURL.absoluteString}
               : [NSNull null],
+          @"images" : [self buildImagesArray:nativeAd.images],
           @"mediaContent" : @{
             @"aspectRatio" : @(nativeAd.mediaContent.aspectRatio),
             @"hasVideoContent" : @(nativeAd.mediaContent.hasVideoContent),
@@ -112,6 +113,19 @@ - (instancetype)init {
       }];
     })
 
+- (id)buildImagesArray:(NSArray<GADNativeAdImage *> *)images {
+  if (images == nil || images.count == 0) {
+    return [NSNull null];
+  }
+  NSMutableArray *formattedImages = [NSMutableArray arrayWithCapacity:images.count];
+  for (GADNativeAdImage *image in images) {
+    if (image.imageURL != nil) {
+      [formattedImages addObject:@{@"scale" : @(image.scale), @"url" : image.imageURL.absoluteString}];
+    }
+  }
+  return formattedImages;
+}
+
 RCT_EXPORT_METHOD(destroy
                   : (NSString *)responseId {
                     if (responseId) {
